awx/ui_next/src/screens/Template/JobTemplateEdit/JobTemplateEdit.test.jsx

```diff
@@ -277,6 +300,7 @@ describe('<JobTemplateEdit />', () => {
       ...mockJobTemplate,
       project: mockJobTemplate.project,
       ...updatedTemplateData,
+      execution_environment: null,
     };
     delete expected.summary_fields;
     delete expected.id;
```

Comment:
@nixocio I think we'll need to do something similar for WFJTs

Reply:
@jakemcdermott, updated. Good catch.  Running e2e tests once more.

Reply:
e2e tests were 🟢